config/locales/modules/charlie.yml

```diff
@@ -18,5 +18,14 @@ en:
         heading: text_page
         body: 3rd module, page 1-1-2
       1-1-3:
-        heading: youtube_page
-        body: 3rd module, page 1-1-3
+        heading: video_page
+        body: |
+          In this video an early years expert explains the importance of the diverse world
+          in the early years foundation stage framework. There are also some tips on how to
+          support children in this area.
+
+          $YOUTUBE[Youtube Test Video](XnP6jaK7ZAY)$YOUTUBE
```

Comment:
Are two videos meant to appear on this page because it is a video template which also has the YT tag in the body?